core/Symbols.h

```diff
@@ -310,6 +311,11 @@ class Symbol final {
         return (flags & Symbol::Flags::CLASS_FINAL) != 0;
     }
 
+    inline bool isClassSealed() const {
+        ENFORCE(isClass());
```

Comment:
can a sealed class be non-abstract? 
What does it mean for it to be non-abstract?